src/components/main/MainBoardCard.jsx

```
@@ -10,19 +10,18 @@ import React from "react";
-        </div>
+    <div
+      className="card main-board-card"
+      onClick={onClick}
+      style={{ cursor: "pointer" }}
+    >
+      <img src={icon} alt={title} className="main-board-card-img" />
+      <div className="card-body">
+        <h5 className="card-title">{title}</h5>
+        <p
+          className="card-text"
+          dangerouslySetInnerHTML={{ __html: description }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)